packages/dart/sshnp_flutter/lib/src/repository/sshnp_partial_params.dart

```diff
@@ -0,0 +1,205 @@
+/// A class which contains a subset of the SshnpParams
```

Comment:
An example of why dartls crashing all the time is really annoying, I moved this file then moved it back and now it is in both places...